frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -295,6 +295,24 @@ function initializeDataset(
     dataSet: dataset.dataSource.id.name,
   });
 
+  // Here we add the originalElementClass property to the segmentation layer if it exists.
```

Comment:
Might be a matter of personal preference, but I think usually comments should short and concise, so `Add the original ...` instead of `Here we add the original ...`. Please drop the `we` from the other comments as well. Apart from that, the comments are written very well :+1:

Reply:
Ok, thanks for that feedback. I'll pay attention to that :slightly_smiling_face: